src/styleguide-components/StyleGuide.js

```diff
@@ -9,13 +9,9 @@ import Ribbon from 'rsg-components/Ribbon';
 import Version from 'rsg-components/Version';
 
 import GlobalStyles from '../components/styles/GlobalStyles';
-import Fonts from '../components/styles/Fonts';
 
 const GlobalStyleGuideStyles = createGlobalStyle`
-  /* System Fonts as used by GitHub */
-  body {
-    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
-  }
+   @import url('https://fonts.googleapis.com/css?family=Open+Sans:400,600,700,800&display=swap');
```

Comment:
aren't we going to get rid of this and instead use a `<link>`?

Reply:
Given this is just our *Stylgeuide* docs I thought it doesn't matter much ( _as they don't use `styled-components` in "production" mode?_ )

Apps should use `<link />` though, I wonder where would be the best place to document that 🤔